src/coreclr/src/pal/src/libunwind_mac/include/elf.h

```diff
@@ -0,0 +1,10 @@
+// This is an incomplete & imprecice implementation
+// It defers to the open source freebsd-elf implementations.
```

Comment:
Some of these files appear to be direct clones of the Windows platform abstraction files I added to make Windows libunwind compile.  I'm not sue how I feel about cloning them again.

I guess it really depends on whether we plan to upstream this.

Reply:
The windows versions of these duplicate files are ifdef'ed _MSC_VER and I didn't want to change them.  We can fix that if we do actually ever upstream these changes.